packages/kuma-gui/src/app/me/sources.ts

```diff
@@ -19,16 +19,18 @@ export const sources = ({ get, set }: Storage) => {
         {
           params: {
             size: 50,
+            format: 'structured',
           },
         },
         app,
         route,
       )
     },
     '/me/:route/:data': async (params) => {
-      const json = JSON.parse(params.data)
-      const res = merge<object>(await get(params.route), json)
-      set(params.route, res)
+      const { global: useGlobal = false, ...json } = JSON.parse(params.data)
```

Comment:
Ideally we wouldn't depend on a "magical" property called `global`, granted super unlikely this is gonna be overwritten by something else, but still. Maybe for the minute we call it `$global`, and maybe longer term we think of another way to hide this. 

We thought of adding a new `me` route, but also we are trying to avoid adding multiple possibly nested datasources.

I wondered if we could wrap the data in two different types of classes ( `new Local({data})` `new Global({data})` and we could tell which type it was with an instance of, but we can't do this as yet because the way DataSink works is that it JSON.stringify's and then JSON.parses the data, which would mean these classes would be lost.

Reply:
We changed the name of this prop to `$global` to avoid any variable name clashing